src/components/delete-generator-faktum-modal/deleteGeneratorFaktumModal.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button, Heading, Modal } from "@navikt/ds-react";
+import { useEffect } from "react";
+import { useQuiz } from "../../context/quiz-context";
+import { useSanity } from "../../context/sanity-context";
+import { generatorFaktumType } from "../generator-faktum-card/GeneratorFaktumCard";
+
+interface IProps {
+  faktumType: generatorFaktumType;
+  isOpen: boolean;
+  handleClose: () => void;
+  delete: (() => void) | undefined;
+}
+
+export function DeleteGeneratorFaktumModal(props: IProps): JSX.Element {
+  const { getAppText } = useSanity();
+  const { faktumType, isOpen, handleClose } = props;
+  const { isLoading } = useQuiz();
+
+  useEffect(() => {
+    if (Modal.setAppElement) {
+      Modal.setAppElement("#__next");
+    }
+  }, []);
+
+  return (
+    <Modal
+      className="modal-container"
+      open={isOpen}
+      onClose={handleClose}
+      closeButton={true}
+      shouldCloseOnOverlayClick={true}
+    >
+      <Modal.Content>
+        <Heading size={"medium"} spacing>
+          {getAppText(`slett-faktum-${faktumType}.modal.tittel`)}
+        </Heading>
+
+        <p>{getAppText(`slett-faktum-${faktumType}.modal.beskrivelse`)}</p>
+
+        <div className="modal-container__button-container">
```

Comment:
Dette kunne kanskje vært en `<nav>`, men er en detalj i det hele. Vi har også en klasse for knapperader i selve søknaden som heter `navigation-container`, kanskje denne også hadde vært fint å bruke her?